bin/run-browser-test-env

```diff
@@ -131,6 +131,9 @@ if truth::is_enabled both_emulators; then
   else
     compose_browser_test "${use_dev}" --profile aws up --no-recreate -d localstack
   fi
+  # Wait for localstack to start. This script also has the side effect of verifying the email
```

Comment:
FYI, there's a large opportunity for cleanup here, but I think a targeted fix makes a lot more sense here. Filed #3497 as tech debt.